runsc/cmd/portforward.go

```diff
@@ -164,8 +165,16 @@ func (p *PortForward) Execute(ctx context.Context, f *flag.FlagSet, args ...any)
 }
 
 // localForward starts port forwarding from the given local port.
-func localForward(ctx context.Context, c *container.Container, localPort int, containerPort uint16) error {
-	l, err := net.Listen("tcp", ":"+strconv.Itoa(localPort))
+func localForward(ctx context.Context, c *container.Container, localSocket string, containerPort uint16) error {
+	var l net.Listener
+	localTCPPort, err := strconv.Atoi(localSocket)
```

Comment:
Is there a possibility that the UDS path could be a number? e.g. a path in the local directory to UDS socket `1234`?

I wonder if we shouldn't have a `--uds` flag that can allow the user to explicitly set their expectation.

Reply:
I just kept the changes as minimal as possible to see if their is interest in the change. Setting a number as path is possible but seems unconventional. I guess when adding a `--uds 1234:80` the equivalent for TCP should be `--tcp 8000:80`?

Reply:
Sure. I'm not super opinionated but I think for backwards compatibility reasons we could just interpret the argument as tcp ports if `--uds` isn't supplied.

e.g. 

TCP:
```
runsc port-forward CONTAINER_ID [LOCAL_PORT:]REMOTE_PORT
```

UDS:
```
runsc port-forward CONTAINER_ID --uds UDS_PATH
```